docs/tutorials/google_earth_engine/part_2/file_upload.rst

```diff
@@ -427,6 +427,8 @@ In this step you will add the logic to validate that the file contained in the Z
           - earthengine-api
           - oauth2client
           - geojson
+          - simplejson
```

Comment:
Is `simplejson` used? Try to stick with the built-in `json` module.

Reply:
it's used starting in the REST API step for handling json errors.